spec/models/book_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+
+RSpec.describe Book, type: :model do
+  context 'validity of Book' do
+    let(:book1) do
+      Book.create(
+        author_name: author_name,
+        title: title,
+        description: description,
+        rating: rating,
+        word_count: word_count
+      )
+    end
+    let(:author_name) { 'Annie Walsh' }
```

Comment:
These let blocks (well, the instance variables defined by them) are redundant.

They're overidden by the ones beginning on line 29. Because they're all in the same scope.

You can confirm this by `puts`ing the `author_name` of both books in one of your tests, I'm pretty sure they'll both be `Kathleen Walsh`.

If you look back to my previous [comment](https://github.com/awalsh3/books/pull/7#discussion_r1167765668) you'll see a working example of how you can use `context` blocks to scope variables. 

However you don't need that here for you current tests.

Try deleting lines 15-18 and seeing what happens. Can you explain why? From there pick a path forward for this spec file and let me know :slightly_smiling_face: